src/server/routes/entity/edition.js

```diff
@@ -122,14 +122,14 @@ router.get(
 	middleware.loadEditionStatuses, middleware.loadEditionFormats,
 	middleware.loadLanguages, middleware.loadRelationshipTypes,
 	(req, res, next) => {
-		const {Publication, Publisher, Work} = req.app.locals.orm;
+		const {EditionGroup, Publisher, Work} = req.app.locals.orm;
 		const propsPromise = generateEntityProps(
 			'edition', req, res, {}
 		);
 
-		if (req.query.publication) {
-			propsPromise.publication =
-				Publication.forge({bbid: req.query.publication})
+		if (req.query['edition-group']) {
```

Comment:
Is any specific reason to use square Bracket instead of the dot notation to retrieve `edition-group` from `req.query`? I think It is creating some inconsistency between dot notation and square bracket. If it is necessary, please mention the reason in the inline comment at that place.

Reply:
`req.query.edition-group` is not valid JS because of the dash between edition and group.
I'll add a comment.